velox/type/Filter.h

```diff
@@ -1819,6 +1823,8 @@ class TimestampRange final : public Filter {
         nullAllowed_ ? "with nulls" : "no nulls");
   }
 
+  bool testInt128(int128_t value) const final;
```

Comment:
Move the implementation here, otherwise it won't be inlined and the `final` is useless

Reply:
Thanks for this suggestion. Updated.